install.sh

```diff
@@ -255,25 +230,24 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   rest="${tarfile#*-}"    # extract string after first '-'
   version="${rest%%-chromeos*}"
                           # extract string between first '-' and "-chromeos"
-
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
-## workaround https://github.com/chromebrew/chromebrew/issues/3305
+# Work around https://github.com/chromebrew/chromebrew/issues/3305
 sudo ldconfig &> /dev/null || true
-echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
-echo -e "${RESET}"
 
-echo_info "Setup and synchronize local package repo..."
-echo -e "${GRAY}"
+echo_out "Setup and synchronize local package repo..."
 
 # Remove old git config directories if they exist
 rm -rf "${CREW_LIB_PATH}"
 
+# Add the CREW_PREFIX bin and musl bin directories to PATH
+export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/share/musl/bin:$PATH
```

Comment:
This only works for install.sh and does not persist with new shell sessions.  Should we be updating `~/.bashrc` also?

Reply:
That won't persist if someone switches shells (eg to fish).

Reply:
Still beside the point.

Reply:
I suppose something could be added in the postinstall of alternative shells to warn users about this. How many alternative shells do we offer?